src/move_basic.cpp

```diff
@@ -199,43 +172,43 @@ MoveBasic::MoveBasic(): tfBuffer(ros::Duration(3.0)),
 {
     ros::NodeHandle nh("~");
 
-    nh.param<double>("min_angular_velocity", minAngularVelocity, 0.05);
-    nh.param<double>("max_angular_velocity", maxAngularVelocity, 1.0);
+    // Velocity parameters
+    nh.param<double>("max_angular_velocity", maxTurningVelocity, 1.0);
```

Comment:
This is totally fine, but you do know once you have dynamic reconfigure, you dont actually need nh.param anymore. The defaut params are set into ros param server from the .cfg file and then  dynamicReconfigCallback is called at initialisation to set the params in the code. Check this out: https://github.com/UbiquityRobotics/avalon/blob/master/avalon_aislemode/src/aisle_path_setter.cpp. Anyway, just recommendation for future improvements :)

Reply:
@JanezCim are you completely sure removing that won't break some legacy setup?

Reply:
nope - that's in the "načeloma" department

Reply:
Hmm.. I do know these parameters get overridden by dynamic reconfigure. I have made all default values the same and would leave like that for convenience, if one decides to use either dynamic_reconfigure or nh.param values.